HanhalOleh/HW7/task7_3.py

```diff
@@ -0,0 +1,7 @@
+def count_letters(word):
```

Comment:
good job
look at this implementation:
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result